package Project_Exception;

public class Exception 

{
	
	public static void main(String args[])
	
	{
	
	try
	{
		int data = 100/0;
	}catch(ArithmeticException ar)
	{
		System.out.println("infinity");
		
	}

}
}
